### ISSUE TITLE
View options: Distraction free is the only option to trigger a snackbar notice

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#66364]
### Description

In the top bar 'Options' ellipsis menu, the first 4 options are related to 'view modes' of the editor:
- Top toolbar
- Distraction free
- Spotlight mode
- Fullscreen more

![Image](https://github.com/user-attachments/assets/1fe0181f-495c-407f-b935-ffa301615b15)

The 'Distraction free' mode is the only one to trigger a snackbar notice. The other ones don't.

![Image](https://github.com/user-attachments/assets/18ecaa91-5465-46ae-bff9-19e272f6e8bf)

The snackbar notice is a little unexpected to me. Also, not sure how the 'Undo' control in the notice is any valuable, given the Options menu is still open and users can just click again there to undo.

On top of that, activating and deactivating these options triggers an audible `speak` message for screen reader users. However, the message for distraction free mode is duplicated. Actually, two different components trigger a message for this same feature. Also, the wording of the message is inconsistent.

Screenshot when dumping all the `speak` messages to the console:

![Image](https://github.com/user-attachments/assets/ab03a85a-ed81-413d-b9f9-e5eb54dce0bd)

## Proposal

Either all these options should trigger a snackbar notice or none.
I'd like to propose to remove the snackbar notice of the Distraction free mode for consistency and simplicity. I'm not sure this snackbar notice adds much value. 

Note: recommended to check what happens when toggling these options via the Command palette.

### Step-by-step reproduction instructions

- Edit a post.
- Go to the top bar > Options.
- Toggle on and off each view option.
- Observe only Distraction free mode triggers a snackbar notice.

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [ ] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure

**Comment by Vrishabhsk** [comment]
Hi @afercia 👋 
- I have raised a `PR` addressing this : https://github.com/WordPress/gutenberg/pull/66405 
- I went ahead and enabled all the `View Options` to show a `Snackbar Notice`
- Since these options, when enabled via the `Command Palette`, show a `notice`, it only makes sense for them to show a `notice` when enabled via the `More Menu` → `View Options`
- It also provides a `visual feedback` to the user in case they are `unfamiliar` with the `option` and do not the `state of the option`

Let me know your thoughts on this. Thanks

**Comment by ntsekouras** [comment]
To add some context, the snackbar was added for distraction free mode because it can be triggered in various conditions besides this toggle. We should definitely keep only one from [the action](https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/store/actions.js#L842).

So, if we want consistency @Vrishabhsk 's approach to add to the others is the way only way to go.

**Comment by mtias** [comment]
I don't think a notice is useful when triggered from the menu. Maybe the command palette should be the one triggering these.

**Comment by afercia** [comment]
> I don't think a notice is useful when triggered from the menu. Maybe the command palette should be the one triggering these.

I'm not sure why the snackbar notice should sometimes appear and sometimes not for similar actions. I'm not sure that's any helpful from an user perspective.  it _may_ seem smart to contributors because we, as developers and designers, we know how these features work under the hood. However, to me, it doesn't make sense from an user perspective and it's just confusing.

Consistency is key. To me, these actions should either always trigger a notice or never.

**Comment by ntsekouras** [comment]
>I'm not sure why the snackbar notice should sometimes appear and sometimes not for similar actions.

This started with distraction free mode because it can be toggled based on other triggers and the user should be notified about the 'automated' change.

>Consistency is key. To me, these actions should either always trigger a notice or never.

If all similar menu items don't trigger a notice can be considered consistent too and it makes sense.

**Comment by afercia** [comment]
> If all similar menu items don't trigger a notice can be considered consistent too and it makes sense.

That's not the case. Activating Distraction free from the menu item does trigger a notice. Activating the other items, doesn't.

![Image](https://github.com/user-attachments/assets/d85c9cb6-ed50-4f6f-86a5-7b4348e540cc)

**Comment by ntsekouras** [comment]
>That's not the case. Activating Distraction free from the menu item does trigger a notice. Activating the other items, doesn't.

Yeah, that's gonna be fixed in the linked PR.
